flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/physical/stream/StreamExecExchange.scala

```diff
@@ -104,6 +106,7 @@ class StreamExecExchange(
           inputTransform,
           partitioner.asInstanceOf[StreamPartitioner[BaseRow]])
         transformation.setOutputType(outputTypeInfo)
+        transformation.setParallelism(ExecutionConfig.PARALLELISM_DEFAULT)
```

Comment:
Could we use `planner.getExecEnv.getParallelism` instead of `ExecutionConfig.PARALLELISM_DEFAULT`? 
Set a -1 on exchange is too subtle to understand it is a `env.Parallelism`.

And it would be better to add a comment on this line, for example, "this enables all the downstream physical nodes to use `inputTransform.getParallelism` to get a good initial parallelism."

Reply:
I think it is better to use ExecutionConfig.PARALLELISM_DEFAULT instead of -1 or env.Parallelism. 
ExecutionConfig.PARALLELISM_DEFAULT means table do not set specific parallelism for  transformations and its parallelism is to be decided by runtime.